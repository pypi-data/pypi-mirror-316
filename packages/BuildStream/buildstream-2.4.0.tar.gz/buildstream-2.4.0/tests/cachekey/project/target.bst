kind: stack
description: |

  This is the main entry point including cases in the cache key test.

depends:
- sources/local1.bst
- sources/local2.bst
- sources/remote1.bst
- sources/remote2.bst
- sources/tar1.bst
- sources/tar2.bst
- elements/build1.bst
- elements/build2.bst
- elements/build3.bst
- elements/compose1.bst
- elements/compose2.bst
- elements/compose3.bst
- elements/compose4.bst
- elements/compose5.bst
- elements/import1.bst
- elements/import2.bst
- elements/import3.bst
- elements/script1.bst
- elements/variables1.bst
