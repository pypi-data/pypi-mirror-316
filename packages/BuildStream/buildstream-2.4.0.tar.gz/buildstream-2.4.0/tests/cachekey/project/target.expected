c4f7317484ebf493139660bd002bd4d62e9fb8c305f7b76e6d814226e8abf37c